packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.types.ts

```diff
@@ -46,7 +46,7 @@ export interface IOverflowSetProps extends React.Props<OverflowSet> {
    * Rendering method for overflow button and contextual menu. The argument to the function is
    * the overflowItems passed in as props to this function.
   */
-  onRenderOverflowButton: IRenderFunction<IOverflowSetItemProps[]>;
+  onRenderOverflowButton: IRenderFunction<IButtonProps[]>;
```

Comment:
oops, didn't notice the array notation. overflowButton is singular, so it'd just be 1 button.

Reply:
I'm confused. Why was it `any[]` and not `any` in the first place?